public/app/features/explore/state/query.ts

```diff
@@ -486,11 +485,11 @@ async function handleHistory(
   queries: DataQuery[],
   exploreId: string
```

Comment:
I think `history` and `exploreId` params are no longer used.

Reply:
Great catch, thanks!